lms/static/js/dashboard/dropdown.js

```diff
@@ -9,9 +9,15 @@ var edx = edx || {};
     // Generate the properties object to be passed along with business intelligence events.
     edx.dashboard.dropdown.toggleCourseActionsDropdownMenu = function(event) {
         // define variables for code legibility
```

Comment:
You didn't add this line but can remove the 'define variables for code legibility' if you want, seems a bit redundant